packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -149,7 +149,9 @@ function getNextDirection(isCurrentSortField, currentSort) {
 export function ColumnHeader({ index, column, sort }) {
 	let header;
 
-	if (sort) {
+	if (column.hideHeader) {
```

Comment:
Does it make sense to test column.label instead of introducing this property ?
I'm thinking about the use cases and I don't think we would want to pass a label to hide it, or to pass an empty label to display it. So if we base the 2 things on the same data instead of having 2, it will avoid those 2 use cases (user props desynchro)
What do you think ?